packages/admin-ui-form/package.json

```diff
@@ -35,6 +35,6 @@
   },
   "dependencies": {
     "@hookform/resolvers": "2.8.8",
-    "react-hook-form": "7.31.1"
+    "react-hook-form": "7.21.0"
```

Comment:
The react.3x builder only accepts engines with node `<=12.x`. This is the last version supported by our platform. We must find a way to upgrade those versions... until there, this change will allow our developers to use the library without the need for yarn resolutions.

Reply:
I'd add a warning in the readme about this. Quite probable this to be updated by someone else in the future willing to do good by updating deps.